packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -239,6 +254,19 @@ export class BlobManager {
     }
 
     public async getBlob(blobId: string): Promise<ArrayBufferLike> {
+        const request = { url: blobId };
+        if (this.tombstonedBlobs.has(blobId) ) {
+            // Note: if a user writes a request to look like it's viaHandle, we will also send this telemetry event
+            this.mc.logger.sendErrorEvent({
+                eventName: "GC_Tombstone_Blob_Requested",
+                url: request.url,
+                viaHandle: true,
```

Comment:
Why add `viaHandle` here? This will always be true right since there is only one entry point to getting blobs.

Reply:
We can get rid of it.